checker/mta_sts.go

```diff
@@ -10,6 +11,36 @@ import (
 	"strings"
 )
 
+// MTASTSResult represents the result of a check for inbound MTA-STS support.
+type MTASTSResult struct {
+	*Result
+	Policy string `json:"policy"`
```

Comment:
nit: quick comment reminder?
```suggestion
	Policy string `json:"policy"` // Whitespace-separated list of hostnames in MTA-STS policy
```

Reply:
Good idea! It actually contains the full text of the policy file, so comment should def be helpful there.